api/src/main/java/com/google/sps/Email.java

```diff
@@ -15,22 +13,22 @@ public Email(User player, String subject, String body) {
   }
 
   public static Email startGameEmail(String gameID, User player) {
-    tempSubject = "You have an invitation!";
-    tempBody = "Welcome to Pictophone!\n\n" + player.getName() + " has invited you to a game! Join here: http://phoebeliang-step.appspot.com/game/" + gameID;
+    String tempSubject = "You have an invitation!";
```

Comment:
What was the reason for this change?

Reply:
this was just a change i made during our 1:1. i committed that change bc i was trying to pull for master to address the authentication problem i was having

Reply:
While sharing one instance for subject and one for body is bad, making instances of the exact same strings isn't the best if you could make them constants instead.

Reply:
right we talked about it, it's on the github issues and i was actually just about to start looking into those

Reply:
sounds good